### PR TITLE
Make examples' javadoc more homogenous and minor html fixes

### PR DESCRIPTION
**Pull request by mziccard**

### CODE REVIEW

**Comment by mziccard** [comment]
Please @jean-philippe-martin take a look

File: gcloud-java-examples/src/main/java/com/google/cloud/examples/compute/ComputeExample.java [hunk 1]
```diff
@@ -83,9 +83,14 @@
  *
  * <p>This example demonstrates a simple/typical Compute usage.
  *
- * <p>See the README for the steps needed for compiling and running the example.
- * <p>Possible command-line arguments are:
- * <pre>{@code [<project_id>]
+ * <p>Steps needed for running the example:
+ * <ol>
+ * <li>log in using gcloud SDK - {@code gcloud auth login}.</li>
+ * <li>compile using maven - {@code cd gcloud-java-examples;
```
Comment (jean-philippe-martin) [inline gcloud-java-examples/src/main/java/com/google/cloud/examples/compute/ComputeExample.java]: The [README](https://github.com/GoogleCloudPlatform/gcloud-java/blob/gcs-nio/gcloud-java-examples/README.md) specifies that you first have to call `mvn install -DskipTests`. We should add this step here.

Though personally I prefer the [DRY](http://c2.com/cgi/wiki?DontRepeatYourself) approach of pointing the user to the README so we don't have multiple copies of the documentation and risk having them inconsistent.

File: gcloud-java-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java [hunk 2]
```diff
@@ -42,16 +42,23 @@
  * <p>This example creates, deletes, gets, and lists zones. It also creates and deletes
  * record sets of type A, and lists record sets.
  *
- * <p>See the README for steps needed for running the example.
- * <p>The command-line arguments can be:
- * <pre>{@code [<project_id>]
+ * <p>Steps needed for running the example:
+ * <ol>
+ * <li>log in using gcloud SDK - {@code gcloud auth login}.</li>
```
Comment (jean-philippe-martin) [inline gcloud-java-examples/src/main/java/com/google/cloud/examples/dns/DnsExample.java]: Same comment as before.

**Comment by jean-philippe-martin** [comment]
Done looking, your turn.

**Comment by mziccard** [comment]
@jean-philippe-martin amended commit according to suggestions, PTAL

**Comment by jean-philippe-martin** [comment]
Thank you, looks good to me!

**Pull request merged**